yarn.lock

```diff
@@ -0,0 +1,3551 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Should we also remove the `yarn.lock` file and update the `package-lock.json` file since it's what we're currently using?

Reply:
I personally prefer yarn since it's faster while installing dependencies but it's possible to do a `yarn install` without problems even if we don't have `yarn.lock`. I removed it and updated `package-lock.json` as suggested. Thanks @maxmellen .